content/algorithms/expected_consensus/_index.md

```diff
@@ -14,81 +14,300 @@ TODO: remove stale .id/.go files
 
 ## Algorithm
 
-Expected Consensus (EC) is a probabilistic Byzantine fault-tolerant consensus protocol. At a high level, it operates by running a leader election every epoch in which, on expectation, a set number of participants may be eligible to submit a block. EC guarantees that these winners will be anonymous until they reveal themselves by submitting a proof of their election, the `ElectionProof`. Each winning miner can submit one such proof per round and will be rewarded accordingly.
+Expected Consensus (EC) is a probabilistic Byzantine fault-tolerant consensus protocol. At a high level, it operates by running a leader election every epoch in which, on expectation, a set number of participants may be eligible to submit a block. EC guarantees that these winners will be anonymous until they reveal themselves by submitting a proof of their election, the `ElectionProof`. Each winning miner can submit one such proof per round and will be rewarded proportionally to its power. **Each miner will submit a proof for election (Leader Election - Step 1) and a proof of storage (Winning PoSt - Step 2).**
```

Comment:
Hmmm...

> Expected Consensus (EC) is a probabilistic Byzantine fault-tolerant consensus protocol.

Is it a consensus protocol or a leader election protocol? Consensus is a very broad term by itself. And calling it "expected" is extremely confusing, because it's never used formally in the description or specification of this protocol... This smells like an inherited term for something that might has been repurposed/rethought/reframed along the way.

Reply:
> Is it a consensus protocol or a leader election protocol? 

It is a consensus protocol that is based on leader election.

(@raulk In genera,l consensus is a broader term. A leader election algorithm is a special way to construct a consensus protocol) 

> I think it's a consensus protocol for (probabilistic/expected) leader election

nope, this does not sound correct 

> Expected Consensus (EC) is a probabilistic Byzantine fault-tolerant consensus protocol. At a high level, it operates by running a leader election every epoch in which, on expectation, a set number of participants may be eligible to submit a block

This sounds good to be. The word "Expected" is explained by the aside "on expectation".